### ISSUE TITLE
Cocoapod wont install Tapjoy on iOS

### ISSUE THREAD

**Issue by arnavvaryani** [mabdelatief/tap_joy_plugin#1]
I can't download the Tapjoy repo when i import it in iOS, shows me the following error - Please fix
`The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/Users/arnav/Desktop/project_demo/ios/Pods/TapjoySDK/TapjoySDK_iOS_v12.7.1/Libraries/Tapjoy.embeddedframework/Tapjoy.framework)`

**Comment by mabdelatief** [comment]
Hi @arnavvaryani , in your Podfile , please comment out this line : 

`use_frameworks!`

so that it will look like this : 

```
target 'Runner' do
# use_frameworks!
  use_modular_headers!
```

**Comment by arnavvaryani** [comment]
Hi! I tried commenting the line out, pod install worked however the app now causes issues with my other dependencies because of doing this.
Can you put s.static_framework = true in your podfile and try instead or do you have any other solution to this?

**Comment by mabdelatief** [comment]
Yes i will try that later tonight and will let you know, 
Thanks for the hint.

**Comment by mabdelatief** [comment]
@arnavvaryani , issue fixed in version 0.0.5 . i will close this issue now, thanks for reporting.
